platform/platform.go

```diff
@@ -127,3 +128,34 @@ func processLookupByServiceName(service string) (bool, []int) {
 
 	return len(pids) > 0, pids
 }
+
+// Returns a map of all the open TCP listening ports and their Pid
+func portPidLookup() map[int]int {
+
+	rx := regexp.MustCompile(`.+\:(\d+)\D*`)
+	portPid := map[int]int{}
+
+	cmd := exec.Command("lsof", "-iTCP", "-sTCP:LISTEN", "-n", "-P", "-T")
+
+	output, err := cmd.Output()
+	if err != nil {
+		fmt.Printf("Failed to list running ports.\n%s\n", err)
+		return portPid
+	}
+
+	scanner := bufio.NewScanner(strings.NewReader(string(output)))
+	for scanner.Scan() {
+		split := strings.Fields(scanner.Text())
+
+		if len(split) == 9 {
+			if matches := rx.FindStringSubmatch(string(split[8])); matches != nil {
+				port, _ := strconv.Atoi(matches[1]) // no err check since regex ensures its a number
+				pid, _ := strconv.Atoi(split[1])    // no err check since regex ensures its a number
+				// return time.Unix(int64(secs), 0)
```

Comment:
can remove this comment

Reply:
Good spot - thanks removed the commented out code